### PR TITLE
Tls sd 020422 rw

### PR DESCRIPTION
**Pull request by plughy2**
SD updates for:

FCS_TLSC_EXT.3.1
FCS_TLSS_EXT.3.1 and 3.2  
FCS_DTLSC_EXT.1.1
FCS_DTLSC_EXT.1.3
FCS_DTLSC_EXT.1.4
FCS_DTLSC_EXT.1.5
FCS_DTLSC_EXT.1.6
FCS_DTLSC_EXT.1.7
FCS_DTLSC_EXT.1.8

### CODE REVIEW

File: ND Supporting Document 2_2.adoc [hunk 1]
```diff
@@ -1812,16 +1903,47 @@ Remark: Some systems might require the presence of the SAN extension. In this ca
...
+. Test 3 [conditional]: The evaluator shall perform the following tests if “present the signature_algorithms_cert extension” is selected:
+[arabic]
+[loweralpha]
+... The evaluator shall examine the Client Hello message and verify it contains the signature_algorihtms_cert extension and the SignatureSchemes match the SignatureSchemes specified in the requirement.
+... The evaluator shall establish a DTLS connection using a certificate chain using each of the SignatureSchemes specified by the requirement. The evaluator shall ensure the signatures used in the certificate chain are consistent with the SignatureScheme being tested.
+[arabic]
+Remark: The server must ensure the DTLS handshake is valid (e.g. certificate key, Certificate Verify message, message formatting, hashes, and signatures), except for the certificate chain using a SignatureScheme not proposed by the client. The selected legacy SignatureScheme shall be as similar to one of the supported SignatureSchemes as possible. For example: 1) ecdsa_sha1 shall not be used if the TOE only supports RSA signatures, 2) ecdsa_sha1 must use a supported ECDSA curve or P-256 if only EdDSA curves are supported.
```
Comment (kr15tyk) [inline ND Supporting Document 2_2.adoc:1936]: @plughy2 Add line break before remark....

**Review by kr15tyk — CHANGES_REQUESTED** [review]
@plughy2 Edited header to match NDcPP , in line 1756 remove 'without mutual authentication'
The paragraph numbering issue will be resolved by the editorial team.

File: ND Supporting Document 2_2.adoc [hunk 2]
```diff
@@ -1604,6 +1604,34 @@ Some TOEs may set up the registration channel before the enablement step is carr
 
 (covered by FCS_TLSC_EXT.1.1 Test 1 and testing for FIA_X.509_EXT.).
 
+==== FCS_TLSC_EXT.3 Extended: TLS Client support for renegotiation
```
[the remark this answers is not in the document]
Comment (plughy2) [reply ND Supporting Document 2_2.adoc]: It wasn't in the the SD but I've added it.

File: ND Supporting Document 2_2.adoc (hunk 1 above)
Comment (plughy2) [reply ND Supporting Document 2_2.adoc:1937]: Added as described

**Review by kr15tyk — APPROVED** [review]

**Pull request merged**